tests/apps/workflowsapp/app.go

```diff
@@ -209,10 +212,21 @@ func main() {
 	w.RegisterWorkflow(HelloTemporalWF)
 	w.RegisterActivity(HelloTemporalAct)
 
+	for i := 0; i < 5; i++ {
+		err = w.Start()
```

Comment:
Any idea on why this needs these retries? Although this works, just wondering what the underlying issue what could cause `w.Start()` to fail would be